packages/nx-plugin-openapi/src/executors/generate-api-lib-sources/executor.ts

```diff
@@ -61,26 +41,35 @@ function generateSources(
   typeMappings: string,
   outputDir: string,
 ): Promise<number> {
+  mkdirSync(outputDir, { recursive: true });
+
   return new Promise((resolve, reject) => {
-    const args = ['generate', '-i', apiSpecPathOrUrl, '-g', generator, '-o', outputDir];
+    const { command, args } = useDockerBuild
+      ? {
+          command: 'docker',
+          args: ['run', '--rm', '-v', `${process.cwd()}:/local`, '-w', '/local', 'openapitools/openapi-generator-cli'],
+        }
+      : { command: 'npx', args: ['openapi-generator-cli'] };
+
+    args.push('generate', '-i', apiSpecPathOrUrl, '-g', generator, '-o', outputDir);
 
     if (additionalProperties) {
-      args.push(...['--additional-properties', additionalProperties]);
+      args.push('--additional-properties', additionalProperties);
```

Comment:
I remember having to use the spread array when I first wrote the plugin because the simpler syntax (this one) just didn't work for some reason.

Did you get the chance to confirm it now works? 🤞

Reply:
yes this did work